cmd/vic-machine/create/create.go

```diff
@@ -767,5 +767,16 @@ func (c *Create) Run(clic *cli.Context) (err error) {
 	executor.ShowVCH(vchConfig, c.Certs.Ckey, c.Certs.Ccert, c.Certs.Cacert, c.Certs.EnvFile, c.Certs.CertPath)
 	op.Info("Installer completed successfully")
 
+	// wait on the logger to finish streaming
+	go func() {
```

Comment:
What happens to this goroutine if we return before this is reached?

Should this be taking `op` as an argument and have a `case <-op.Done():` so that this gets cleaned up if `op` is canceled (which I _think_ happens just before we exit)?